HW_2/task2.py

```diff
@@ -0,0 +1,31 @@
+# Задача 2
+
+# Напишите программу, которая получает целое число и возвращает его шестнадцатеричное
+# строковое представление.
+# Функцию hex используйте для проверки своего результата.
+
+
+def dec_to_hex(dec):
+    # Здесь сохраняем итоговое шестнадцатеричное представление числа.
+    hex_num = ''
+    # Создаем строку с шестнадцатеричным представлением числа
+    hex_digits = '0123456789ABCDEF'
+
+    while dec > 0:
+        # вычисляем последовательные остатки от деления числа `dec` на основание 16
+        remainder = dec % 16
+        # Символ шестнадцатеричной цифры, соответствующий текущему остатку, запоминается на каждой итерации
+        hex_digit = hex_digits[remainder]
```

Comment:
Если переменная используется единожды, то объявлять ее не обязательно.
```suggestion
        # Символ шестнадцатеричной цифры, соответствующий текущему остатку, запоминается на каждой итерации
        hex_digit = hex_digits[dec % 16]
```